src/components/Dashboard/TodoList.vue

```diff
@@ -1038,6 +1041,14 @@ export default {
     /** Closes current panel or opens new panel. */
     togglePanel (index: number) {
       this.panel = (this.panel === index) ? null : index
+    },
+
+    /** Determine the Annual Report Due date for a given filing. */
+    arDueDate (filingYear: number, foundingDate: Date) {
+      let dueDate = new Date(foundingDate)
```

Comment:
Can be const, since you don't reassign this property.

Reply:
good catch, thanks!